rq/job.py

```diff
@@ -716,7 +716,7 @@ def dependents_key(self):
 
     @property
     def dependencies_key(self):
-        return '{0}:{1}:dependencies'.format(self.redis_job_namespace_prefix, self.id)
+        return '{0}{1}:dependencies'.format(self.redis_job_namespace_prefix, self.id)
```

Comment:
I'd rather not change this because this would break existing keys.

Reply:
Reverted this change